packages/storybook_flutter/lib/src/knobs/knob_list_tile.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+class KnobListTile extends StatelessWidget {
+  const KnobListTile({
+    Key? key,
+    required this.enabled,
+    required this.nullable,
+    required this.onToggled,
+    this.title,
+    this.subtitle,
+    this.isThreeLine = false,
+  }) : super(key: key);
+
+  final Widget? title;
+  final Widget? subtitle;
+  final bool enabled;
+  final bool nullable;
+  final ValueChanged<bool> onToggled;
+  final bool isThreeLine;
+
+  @override
+  Widget build(BuildContext context) => SwitchListTile(
```

Comment:
Let's rather hide the switch if knob is not nullable, instead of just disabling it – it would look cleaner.